### PR TITLE
Muscles's UI updates for the day

### PR DESCRIPTION
**Pull request by nmalfroy**
If you go through commit by commit, it's a bunch of small updates/fixes

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -1186,12 +1186,26 @@ const driver = neo4j.driver(
...
+             n.definition as Definition,
+             n.uri as URI, 
+             collect(n.label) as Synonyms,
+             collect(s.label) as Subclasses
+      limit 10
+    `
+    const res = await session.run(query)
     await session.close()
-    console.log(res);
+    console.log(res)
```
Comment (s-rubenstein) [inline src/libs/ajax.js:1208]: We should probably clear out the console.logs

File: src/pages/ontology/OntologyModal.js [hunk 2]
```diff
@@ -71,18 +71,18 @@ const AboutOntologySearch = () => {
 
 const OntologySearch = ({ searchTerm }) => {
   const [data, setData] = useState(undefined)
+  const [submittedSearchTerm, setSubmittedSearchTerm] = useState(undefined)
```
Comment (s-rubenstein) [inline src/pages/ontology/OntologyModal.js:74]: Rather than a new piece of state, I think we can use `prevState` here?

Comment (s-rubenstein) [reply src/pages/ontology/OntologyModal.js:74]: Talked offline

File: src/pages/ontology/OntologyContainer.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const OntologyContainer = ({ title, setModalOpen, children }) => {
   return h(FooterWrapper, [
     h(TopBar, { title, href: Nav.getLink('root') }, [
       div({ style: { flexGrow: 1 } }),
-      div({ onClick: () => setModalOpen(true) }, ['hello'])
+      div({ onClick: () => setModalOpen(true), style: { marginRight: 10 } }, ['hello'])
       // TODO ontology search
```
Comment (Shakespeared) [inline src/pages/ontology/OntologyContainer.js]: I think we can take out this TODO as well

**Review by Shakespeared — APPROVED** [review]
other than removing console logs etc, LGTM!

**Pull request merged**